### PR TITLE
Hotfix/disable uninstall loop external data

### PR DESCRIPTION
**Pull request by martinyde**
The update hook in this in this pull request should be run before any further pull requests are merged, since it might require some code that will be deleted in the future.

### CODE REVIEW

File: loopdk.install [hunk 1]
```diff
@@ -125,3 +125,45 @@ function loopdk_translation_update() {
...
+  node_types_rebuild();
+  menu_rebuild();
+
+  // Uninstall module.
+  if (module_exists('loop_external_data')) {
+    module_disable(array('loop_external_data'));
+    drupal_uninstall_modules(array('loop_external_data'));
+  }
+
+  // CLeanup deleted fields
```
Comment (rimi-itk) [inline loopdk.install]: Please spell check "CLeanup".

File: loopdk.install [hunk 2]
```diff
@@ -125,3 +125,45 @@ function loopdk_translation_update() {
...
+  drupal_cron_run();
+}
```
Comment (rimi-itk) [inline loopdk.install]: Add missing newline.

File: loopdk.install [hunk 3]
```diff
@@ -125,3 +125,45 @@ function loopdk_translation_update() {
...
+  menu_rebuild();
+
+  // Uninstall module.
+  if (module_exists('loop_external_data')) {
+    module_disable(array('loop_external_data'));
+    drupal_uninstall_modules(array('loop_external_data'));
+  }
+
+  // CLeanup deleted fields
+  drupal_cron_run();
```
Comment (rimi-itk) [inline loopdk.install]: This looks fishy. Cron will potentially do a lot more that cleaning up deleted fields. Can we find a more dedicated function to call?

File: loopdk.install [hunk 4]
```diff
@@ -125,3 +125,45 @@ function loopdk_translation_update() {
...
+ * Delete nodes and content types for leaf, tree and index.
+ */
+function loopdk_update_7002() {
+  $types = array(
+    'leaf',
+    'index',
+    'tree'
+  );
+  foreach($types as $type) {
+    // First, delete nodes
```
Comment (rimi-itk) [inline loopdk.install]: Please run `phpcs --standard=Drupal` on the code.

File: loopdk.install (hunk 3 above)
Comment (martinyde) [reply loopdk.install]: Removed the cron_run function. We just let Drupal do it's cron cleanup as part of the ordinary flow.

**Pull request merged**